packages/core/src/sass/app/modules/reset-password.scss

```diff
@@ -22,6 +22,7 @@
         margin-bottom: 3.3rem;
     }
     &__subtext {
+        text-align: center;
```

Comment:
Can we pass it to the `Text` component as a prop instead of adding it to css styles?

Reply:
It's it required in two places [signup](https://github.com/binary-com/deriv-app/blob/a68befca691c9bf536b3c1b99f8b686d9c61ecf1/packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx#L163) and [reset-password](https://github.com/binary-com/deriv-app/blob/a68befca691c9bf536b3c1b99f8b686d9c61ecf1/packages/core/src/App/Containers/ResetPasswordModal/reset-password-modal.jsx#L122) both of them will need the text to be aligned to the center. Instead of writing `align` property for each one I guess it is better to include it within the css styles

Reply:
Hmm, I still think it would be better to pass it for each component, as we are already passing stuff like size to it.
Otherwise, it would make it harder to handle our styles in the future as some of them are passed as props and some are in the stylesheets.